src/function/debounce.spec.ts

```diff
@@ -17,6 +17,21 @@ describe('debounce', () => {
     expect(func).toHaveBeenCalledTimes(1);
   });
 
+  it('should not immediately call the function when debounce time is 0', async () => {
```

Comment:
Is this actually what we offer with debounce? I thought this was implementation detail.